hbase/upload_new_phoenix_version.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -e
+
+VERSION=${1:?"Missing version number argument (arg 1)"}
+NEXUS_USER=${2:?"Missing Nexus username argument (arg 2)"}
+
+read -r -s -p "Nexus Password: " NEXUS_PASSWORD
+echo ""
+
+# https://stackoverflow.com/questions/4632028/how-to-create-a-temporary-directory
+# Find the directory name of the script
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# the temp directory used, within $DIR
+WORK_DIR=$(mktemp -d -p "$DIR")
+
+# check if tmp dir was created
+if [[ ! "$WORK_DIR" || ! -d "$WORK_DIR" ]]; then
+  echo "Could not create temp dir"
+  exit 1
+fi
+
+# deletes the temp directory
+function cleanup {
+  rm -rf "$WORK_DIR"
+}
+
+# register the cleanup function to be called on the EXIT signal
+trap cleanup EXIT
+
+cd "$WORK_DIR" || exit
+
+src_file=phoenix-$VERSION-src.tar.gz
```

Comment:
You could argue it does not cost us anything to also download the pre-build version and it could save us the day in the future (e.g. quickly pull it on your machine and test something). But that's 100% speculative ;)

Reply:
Yeah we'll likely improve this whole "build from source" workflow at some point in the future (how to apply custom patches, how to test upcoming versions, how to track where we pulled stuff from in a machine-readable way etc.). I'd say it's out of scope for now, the main goal was just about building Phoenix from source (and testing it) instead of pulling the binaries.